addons/mrp/models/stock_orderpoint.py

```diff
@@ -42,6 +42,21 @@ def _compute_show_bom(self):
         for orderpoint in self:
             orderpoint.show_bom = orderpoint.route_id.id in manufacture_route
 
+    def _get_min_qty_to_order(self):
+        """ Calculates the minimum quantity that can be ordered according to the qty and UoM of the BoM
+        """
+        self.ensure_one()
+        min_qty_order_possible = super()._get_min_qty_to_order()
+        if min_qty_order_possible:
+            return min_qty_order_possible
+        elif self.route_id:
```

Comment:
Why limit the result when the `route_id` is set ? The `route_id` is only when you want to force/limite the route.
(same for purchase)

Reply:
I'm not sure I understand. The idea of taking `route_id` is because it represents the "Preferred Route" column of the replenishment page, which can be easily changed by the user. 
Do you mean that in case there are no preferred route, I could look at all the rules of the orderpoint where `action = 'manufacture'` and take for example the one with the smallest minimum quantity?

Reply:
Well, the preferred route is optional, it just change the way how to get the `rules_ids` (`compute_rules_ids`). But it is the `rules_ids` which determines if we are in the case of 'manufacture' or 'purchase'. If you not set the route_id, some rules_ids will be still appliqued (determine in `_search_rule`). But maybe we need this only if we force the route_id ?

Reply:
Ok, now I understand that I didn't understand well rule_ids.
You comment makes perfect sense now :-)